examples/vertx-opentracing/README.adoc

```diff
@@ -0,0 +1,50 @@
+= Example Vertx Application instrumented using Opentracing API
+
+This example provides a simple order management system. When a user places an order, the Order Manager communicates with:
+
+- Account Manager to verify the account id is known
+- Inventory Manager to check there is enough stock
+- Once the order is confirmed and acknowledged to the user, it will be stored in the Order Log and the Inventory Manager
+will be updated.
+
+NOTE: Once the end user response has been returned, to confirm the order, the notification to the Order Log and Inventory
+Manager is performed via publishing a message on a shared destination.
+
+The other operation that can be performed by an end user is to list the current orders associated with an account id.
+
+== Run
+
+First step is to start up the APM server. Then open a console window and set the HAWKULAR_APM_URI,
+HAWKULAR_APM_USERNAME and HAWKULAR_APM_PASSWORD environment variables.
+
+From this folder, run:
+
+[source,shell]
+----
+$ mvn clean install
+$ java -jar target/hawkular-apm-examples-vertx-opentracing-fat.jar
```

Comment:
If there were docker image, it would be easy for users to bind it to the different port.